### DISCUSSION TITLE
Add Support for Monitoring AirPlay Streams Initiated from Other Devices

### DISCUSSION THREAD

**Discussion by daydy16** [music-assistant/support#2021] [Feature requests and ideas]
I am a user of your software and I find it very useful for managing and playing my music. I appreciate the existing support for streaming music to AirPlay speakers. However, I have noticed that there is currently no support for monitoring or controlling AirPlay streams that are initiated from other devices, such as an iPhone.

I often play music using AirPlay from my iPhone or even Siri and it would be very beneficial if I could see and control this playback directly from Music Assistant. This would provide a more seamless and integrated experience for users who utilize AirPlay to stream their music from various devices. Additionally, it would cater to less tech-savvy users who prefer not to use an additional user interface or app for this functionality.

Thank you for considering my request. I look forward to any updates on this potential feature.

**Comment by OzGav** [comment]
For Airplay 1 it is not possible. It would be possible to implement it for airplay 2 for basic control of real apple devices like homepod and ATV but  for the development effort the devs feel that it is not worth the trouble. They would rather focus on high quality audio playback to airplay targets (not limited to apple gear).
